doc/src/aboutus.rst

```diff
@@ -24,23 +24,36 @@ Financial and Infrastructure Support
   * in 2012 for 6 students (`GSoC 2012 <https://github.com/sympy/sympy/wiki/Gsoc-2012-report>`_)
   * in 2013 for 7 students (`GSoC 2013 <https://github.com/sympy/sympy/wiki/Gsoc-2013-report>`_)
   * in 2014 for 10 students (`GSoC 2014 <https://github.com/sympy/sympy/wiki/Gsoc-2014-report>`_)
+  * in 2015 for 7 students (`GSoC 2015 <https://github.com/sympy/sympy/wiki/Gsoc-2015-report>`_)
+  * in 2016 for 9 students (`GSoC 2016 <https://github.com/sympy/sympy/wiki/GSoC-2016-report>`_)
+
+  Of these, we would like to thank these organizations for hosting GSoC students under their
+  umbrella organizations:
+
+  * `Python Software Foundation (PSF) <https://www.python.org/psf/>`_ has hosted
+    various GSoC students over the years:
+
+    * 3 GSoC 2007 students (Brian, Robert and Jason)
+    * 1 GSoC 2008 student (Fredrik)
+    * 2 GSoC 2009 students (Freddie and Priit)
+    * 4 GSoC 2010 students (Aaron, Christian, Matthew and Øyvind)
+    * 7 GSoC 2015 students
```

Comment:
Looking at the GSoC 2015 report https://github.com/sympy/sympy/wiki/Gsoc-2015-report it only says 5 students. Should it be 5 then? Or is that report missing something?
BTW this number here represents the total number of GSoC students (participated under SymPy + some umbrella organisation).